src/test/ui/trait-bounds/unsized-bound.stderr

```diff
@@ -0,0 +1,234 @@
+error[E0277]: the size for values of type `B` cannot be known at compilation time
+  --> $DIR/unsized-bound.rs:2:12
+   |
+LL | trait Trait<A> {}
+   |             - required by this bound in `Trait`
+LL | impl<A, B> Trait<(A, B)> for (A, B) where A: ?Sized, B: ?Sized, {}
+   |         -  ^^^^^^^^^^^^^ doesn't have a size known at compile-time
+   |         |
+   |         this type parameter needs to be `std::marker::Sized`
+   |
+   = note: required because it appears within the type `(A, B)`
+help: consider removing the `?Sized` bound to make the type parameter `Sized`
+   |
+LL | impl<A, B> Trait<(A, B)> for (A, B) where A: ?Sized, {}
+   |                                                   --
```

Comment:
not this PRs problem, but we really should move to something that makes it clear it's not the two marked chars, but the removed chars between them

Reply:
Addressed in #86532